### DISCUSSION TITLE
Keyboard Shortcuts?

### DISCUSSION THREAD

**Discussion by nickheniser** [usebruno/bruno#1079] [Q&A] [answered]
Does anyone know where there might be a list of keyboard shortcuts to use with bruno?

I am specifically looking for how to collapse and expand json in the body of a request/response for easier viewing.

**Comment by n00o** [comment, marked as answer]
You can collapse all and expand all for JSON bodies. Here are the hotkeys:
* `Ctrl-Y` - Collapse All
* `Ctrl-I` - Expand All

It would probably be good to add buttons to do those 2 functions without hotkeys. If you want to make a issue for it then I will try to add it.

There is a draft for a keyboard shortcut windows but it isn't fully completed yet. Seems like a very useful feature. The link for it is below.
* https://github.com/usebruno/bruno/pull/924

**Reply by nickheniser** [reply]
Thanks!  https://github.com/usebruno/bruno/issues/1087

**Reply by ClaireMaud** [reply]
How would I know about this, if not from there? :( Keybindings settings so poor, and no customization :(

**Comment by Khyme** [comment]
@n00o cmd Y makes bruno crash on my mac M2

**Comment by j10sv2** [comment]
@n00o - was looking for a way to collapse json, found this discussion, hit cmd+Y, Bruno crashed.  Then I saw another mac user commented the same.  Thought I would report that its still happening.

**Reply by Khyme** [reply]
Ha, now it works for me!
